repose-aggregator/components/filters/header-translation-filter/src/main/scala/org/openrepose/filters/headertranslation/HeaderTranslationFilter.scala

```diff
@@ -101,3 +108,9 @@ class HeaderTranslationFilter @Inject()(configurationService: ConfigurationServi
     initialized = true
   }
 }
+
+object HeaderTranslationFilter {
+  private final val DefaultConfig = "header-translation.cfg.xml"
+  private final val SchemaFileName = "/META-INF/schema/config/header-translation.xsd"
+  private final val QualityReqex = ";q=\\d[^;]*".r
```

Comment:
I wanted to enforce as much as possible, but once you get past the first digit, it gets a little nuts.  After the first digit, there might be a period, and if there's a period, there might be up to three digits, of which must be zeros if the first digit was a "1".  This seemed like the most I could easily check without going crazy.  Still, I see your point.  Actually, I really should have made the \d be [01] or not even care and just go for the [^;].  Hmmm.....